backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/map/BpkIconMapMarker.kt

```diff
@@ -55,8 +58,8 @@ fun BpkIconMapMarker(
     state: MarkerState = rememberMarkerState(),
     tag: Any? = null,
     visible: Boolean = true,
-    zIndex: Float = 0.0f,
-    onClick: (Marker) -> Unit = {},
+    zIndex: Float? = null,
+    onClick: (Marker) -> Boolean = { false },
```

Comment:
we need to allow consumers to consume the event - for example on hotels we do not want to centre the map when tapping on a marker